core/src/main/java/org/javarosa/core/io/StreamsUtil.java

```diff
@@ -7,10 +7,18 @@
 
 public class StreamsUtil {
 
-    private StreamsUtil() {
-        // private constructor
+    public static byte[] getStreamAsBytes(InputStream is) throws IOException {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+        try {
+            StreamsUtil.writeFromInputToOutput(is, baos);
+            return baos.toByteArray();
+        } finally {
+            baos.close();
```

Comment:
I once forgot to close something and it broke a lot of my logic. Based on that experience I think it is probably sorta important to crash if something fails to close. So I'd vote for leaving this how it is, but no especially well informed reason for this opinion